clients/python/Makefile

```diff
@@ -13,7 +13,7 @@ clean:
 
 .PHONY: build-mr
 build-mr:
-	cd ../../ && make image/build
+	cd ../../ && IMG_REGISTRY=docker.io IMG_VERSION=latest make image/build
```

Comment:
can you kindly make this as a default?

ie:
```makefile
IMG_REGISTRY ?= docker.io
IMG_VERSION ?= latest
```

at the top? wdyt?

Reply:
good one, will do.